tap_snowflake/sync_strategies/common.py

```diff
@@ -142,20 +142,43 @@ def generate_select_sql(catalog_entry, columns):
     return select_sql
 
 
-def generate_copy_sql(select_sql, prefix, temp_s3_upload_folder=None, temp_s3_creds=None):
```

Comment:
previously, this function supported generating copy query for unloading to external and internal stage. Now, divided the functions into two. In the future, we will move to using internal stage unload for default snowflake import.